src/epgsql.erl

```diff
@@ -44,11 +44,12 @@
 
 -type sql_query() :: iodata().
 -type host() :: inet:ip_address() | inet:hostname().
+-type password() :: string() | fun().
```

Comment:
Can you, please, define `fun()` as `fun( () -> iodata() )`? Also, password can be not only a string but binary or any iodata as well

Reply:
Updated. Since `iodata()` is `iolist() | binary()` I did not add `binary()` explicitly.

Reply:
@onno-vos-kivra perfect, thanks!